google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -630,6 +630,44 @@ def patch_file bucket_name,
           end
         end
 
+        ##
+        # Moves object from source to destination path within bucket
```

Comment:
Please remove the keyword `object` and use `file` instead.
```suggestion
        # Moves file from source to destination path within bucket
```